frontend/src/container/GroupedNotificationItem.jsx

```diff
@@ -0,0 +1,209 @@
+      })
+
+      if (numberOfContents !== 1) notificationDetails.url = undefined
+      return notificationDetails
+    }
+  }
+
+  handleClickGroupedNotification = (e, notification) => {
+    const { props } = this
+
+    if (notification.type.includes(GROUP_NOTIFICATION_CRITERIA.CONTENT)) {
+      notification.group.forEach(notification => props.onClickNotification(e, notification.id, {
+        url: PAGE.CONTENT(notification.content.type === TLM_SUB.COMMENT
+          ? notification.content.parentId
+          : notification.content.id)
+      }))
+    } else {
+      props.dispatch(updateNotification(notification.id, notification.group))
+    }
+  }
+
+  render () {
+    const { props } = this
+    const { notification, user } = props
+    const notificationDetails = this.getGroupedNotificationDetails(notification)
+    if (Object.keys(notificationDetails).length === 0) return null
+
+    const numberOfWorkspaces = uniqBy(notification.group.map(notification => notification.workspace), 'id').length
+    const numberOfAuthors = notification.author.length
+    const readStatus = notification.group.map(notification => notification.read)
+      .reduce((acc, current) => acc && current)
+
+    return (
+      <Link
+        to={notificationDetails.url || '#'}
+        onClick={(e) => this.handleClickGroupedNotification(e, notification)}
+        className={classnames('notification__list__item',
+          { itemRead: readStatus, isMention: notificationDetails.isMention }
+        )}
+        key={notification.id}
+      >
+        <div className='notification__list__item__text'>
+          {numberOfAuthors <= 2
+            ? (
+              <div className='notification__list__item__text__avatars'>
+                <Avatar
+                  size={numberOfAuthors === 2 ? AVATAR_SIZE.MINI : AVATAR_SIZE.SMALL}
+                  apiUrl={FETCH_CONFIG.apiUrl}
+                  user={notification.author[0]}
+                />
+                {numberOfAuthors === 2 && (
+                  <Avatar
+                    size={AVATAR_SIZE.MINI}
+                    apiUrl={FETCH_CONFIG.apiUrl}
+                    user={notification.author[1]}
+                    customClass='notification__list__item__text__avatars__second'
+                  />
+                )}
+              </div>
+            ) : (
+              <Icon
+                customClass='notification__list__item__text__usersAvatar'
+                icon='fas fa-fw fa-users'
+                color='#fdfdfd' // INFO - GB - 2021-08-26 - offWhite color
+                title={notification.author.map(author => author.publicName)}
+              />
+            )}
+          <span
+            className='notification__list__item__text__content'
+            dangerouslySetInnerHTML={{
+              __html: (
+                notificationDetails.text + ' ' +
+                `<span title='${escapeHtml(formatAbsoluteDate(notification.created, user.lang))}'>` +
+                '</span>'
```

Comment:
is this span empty on purpose?

Reply:
I don't think so, I just took the code that already existed, I will update it.